examples/nlp/lasertagger/configs/lasertagger_config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "attention_probs_dropout_prob": 0.1,
+  "hidden_act": "gelu",
+  "hidden_dropout_prob": 0.1,
+  "hidden_size": 768,
+  "initializer_range": 0.02,
+  "intermediate_size": 3072,
+  "max_position_embeddings": 512,
+  "num_attention_heads": 12,
+  "num_hidden_layers": 12,
+  "type_vocab_size": 2,
+  "vocab_size": 28996,
+  "use_t2t_decoder": false,
+  "decoder_num_hidden_layers": 1,
+  "decoder_hidden_size": 768,
+  "decoder_num_attention_heads": 4,
+  "decoder_filter_size": 3072,
+  "use_full_attention": false
+}
```

Comment:
What are those parameters and how do you load them? 

Can you please use module configuration and standard `import_from_config()` method?

Reply:
I load it here: https://github.com/NVIDIA/NeMo/pull/751/files#diff-339de8cc968a0ba1ee01e3bea499ead4R163

Reply:
will change it to `import_from_config()`

Reply:
yeah, I saw it - and commented too ;)

Reply:
ok removed the config altogether, and added everything to the main file for consistency.

Reply:
ok, this is surprising... I was honestly expecting the opposite... ;)

may I just ask why? aren't those params useful for experimentation anymore?

Reply:
Oh I added them in main file. They are still there. Reading from config and then again overriding them through args was unnecessary I guess. So just set them through args.

Reply:
hmmm, not sure if I understand... the overwrite mechanism was added to enable to override _some_ of the parameters (e.g. filenames) that are passed straight from the command line args.

Reply:
Yeah, but with this change, don't need to read the configuration anymore. The original configurations are set as defaults in argparse

Reply:
From line 13-14 are the LaserTagger specific configs(Transformer Decoder specific), rest were BERT specific which were never used in the first place.